Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -41,7 +41,8 @@
     "# %env https_proxy=http://<proxy_host>:<port>\n",
     "# Install required packages (Kubeflow Pipelines and Katib SDK).\n",
     "import sys\n",
-    "!{sys.executable} -m pip install \"pyarrow>7.0.0\" \"kfp>=1.8.0,<=1.8.22\" kubeflow-katib==0.16.0"
+    "# !{sys.executable} -m pip install \"pyarrow>7.0.0\" \"kfp>=1.8.0,<=1.8.22\" kubeflow-katib==0.16.0\n",
```

Comment:
I think that we don't need to install kfp in the example itself. It should be available in the notebook by default.

Reply:
Agreed. I'll remove it.

Reply:
Done

Reply:
It actually was worse then we don't need to - it was installation of the previous version, so if user uncomment that, it could brake whole example.

p.s. it was also already covered by my point from the comment above.:

> Commenting code make sense only, if it is expected that user will uncomment it for any reason (like setting proxy), but old code should just be deleted.